.travis.yml

```diff
@@ -3,6 +3,7 @@ env:
     - CC_TEST_REPORTER_ID=38776e3c20a07980a0cc3c66c7519e9b42542a8f07d5bcc9d7498d02cc00a612
     - WCRS_REGISTRATION_EXPIRES_AFTER=3
     - WCRS_REGISTRATION_RENEWAL_WINDOW=3
+    - WCRS_REGISTRATION_GRACE_WINDOW=5
```

Comment:
I agree with you that we've been inconsistent about stubbing env vars vs putting them in the Travis config and need  to decide on one. I would prefer stubbing as it means the tests should still run as expected in any environment – for example, if we changed our CI or ran the suite on a different machine without setting up the env vars. Maybe something to log as an issue for standardising down the line?

Reply:
I would prefer stubbing as well. I'll add an issue, but as I found this by accident rather than design I'd want to stick with leaving the Travis config, and log an issue to remove them all from travis and deal with the fall out then.

Reply:
Yep, agreed!

Reply:
Issue #323 has been created for this